feat: ignore all Manifest files

frankebel opened the pull request:
Starting with Julia 1.11, version-specific `Manifest.toml` files can exist side-by-side.

Closes #462

frankebel committed:
feat: ignore all Manifest files.

Starting with Julia 1.11, version-specific `Manifest.toml` files can
exist side-by-side.

oxinabox commented:
This needs fixing on 1.10.6
probably some reference files need to be regenerated

frankebel commented:
> This needs fixing on 1.10.6 probably some reference files need to be regenerated

Updated the reference files. Also changed the code to ignore `/docs/Manifest*.toml`.

oxinabox commented:
thank you. sorry for the the delay.
